### ISSUE TITLE
Update com.netflix.servo:servo-core dependency

### ISSUE THREAD

**Issue by HuppiFluppi** [Netflix/Hystrix#1496]
I know there was already an issue #1185 for upgrading hystrix-servo-metrics-publisher to use most recent servo library.
As the newest version depends on Java8 this was shifted to version 2.0.

However i wonder if we can update to the last Java6 version of servo (which i guess would be 0.10.0).

Background is a problem with the guava version(14) in the now used servo lib (7.5) which would probably vanish with the update.

TL;DR: Updating servo-core dependency to 0.10.0

**Comment by mattrjacobs** [comment]
Merged in #1498
